### DISCUSSION TITLE
Grow/Shrink Window Vertically and Horizontally

### DISCUSSION THREAD

**Discussion by prasskaly** [rxhanson/Rectangle#575] [Ideas]
Can we grow or shrink window vertically only and horizontal only ?

**Comment by rxhanson** [comment]
I’ll think about this one. Are you looking for grow/shrink in both directions or just one? I.E. vertical adjusts both top and bottom with one command or just top with one command and bottom with another

**Comment by prasskaly** [comment]
Vertically adjusting both top and bottom with one command would be the ideal solution. 
If the window is already touching the screen edge(or Menu bar) at the top , then this solution will grow the bottom side only and vice versa. This behavior is the same as in "Smaller" and "Larger"  already featured in the app where the window border closer to the edge of the display does not grow anymore and only the other side grows.
